src/app/shared/components/template/components/button/button.component.scss

```diff
@@ -67,6 +68,52 @@ ion-button[data-param-style~="card"] {
   }
 }
 
+.button-container[data-variant~="card-portrait"] {
+  background-color: white;
+  color: var(--ion-color-primary);
+  font-size: var(--buttons-font-size-small);
+  font-weight: var(--font-weight-bold);
+  border: 1px solid var(--ion-color-gray-light);
+  border-radius: var(--ion-border-radius-secondary);
+  min-height: var(--buttons-short-height);
+  filter: drop-shadow(var(--ion-default-box-shadow));
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1.5rem 0.5rem 0.5rem;
+  // TODO: make a variable, shared with task card
+  width: 165px;
```

Comment:
nit(non-blocking) another fairly non-standard size, given most sizing will scale in increments of 16px (or sometimes 8px/4px) 160px might make a more sense